src/test/components/__snapshots__/MarkerChart.test.js.snap

```diff
@@ -51,6 +51,35 @@ Array [
     "set lineWidth",
     1,
   ],
+  Array [
```

Comment:
I don't know if that's "correct" but this is expected to draw more markers, and that the existing ones are shifted to the left because the range is larger. So this seems consistent.